.github/workflows/e2e-wide-ep-accelerator-gke.yaml

```
@@ -0,0 +1,260 @@
+          git checkout pr-"$PR_OR_BRANCH"
+
+      - name: Checkout branch
+        if: steps.check_pr.outputs.is_pr == 'false'
+        run: git checkout "$PR_OR_BRANCH"
+
+      - name: Authenticate to Google Cloud
+        id: auth
+        uses: google-github-actions/auth@b7593ed2efd1c1617e1b0254da33b86225adb2a5
+        with:
+          credentials_json: ${{ secrets.GKE_SA_KEY }}
+
+      - name: Set up gcloud CLI and kubectl
+        uses: google-github-actions/setup-gcloud@cb1e50a9932213ecece00a606661ae9ca44f3397
+        with:
+          project_id: ${{ env.GCP_PROJECT_ID }}
+          install_components: 'kubectl,gke-gcloud-auth-plugin'
+
+      - name: Get GKE credentials
+        run: |
+          gcloud container clusters get-credentials "${{ env.GKE_CLUSTER_NAME }}" --zone "${{ env.GKE_CLUSTER_ZONE }}"
+
+      - name: Install dependencies
+        run: |
+          cd guides
+          ./prereq/client-setup/install-deps.sh | tee ~/install-deps.log
+
+      - name: Create namespace
+        run: |
+          kubectl create namespace "${NAMESPACE}" || echo "Namespace already exists"
+
+      - name: Create llm-d-hf-token secret
+        run: |
+          kubectl create secret generic llm-d-hf-token \
+            --from-literal="HF_TOKEN=${{ secrets.HF_TOKEN }}" \
+            --namespace "${NAMESPACE}" \
+            --dry-run=client -o yaml | kubectl apply -f -
+
+      - name: Deploy model server
+        run: |
+          cd guides/wide-ep-lws
+          echo "Deploying Model Server..."
+          kubectl apply -k ./manifests/modelserver/gke -n ${NAMESPACE} | tee ~/wide-ep-deployment.log
+          echo "---------------------------------------" >> ~/wide-ep-deployment.log
+
+      - name: Deploy InferencePool
+        shell: bash
+        run: |
+          cd guides/wide-ep-lws
+          echo "Deploying InferencePool..."
+          helm install deepseek-r1 \
+          --namespace ${NAMESPACE} \
+          --set "provider.name=gke" \
+          --set "inferencePool.apiVersion=inference.networking.k8s.io/v1" \
+          --set "inferenceExtension.monitoring.gke.enable=true" \
+          -f inferencepool.values.yaml \
+          oci://us-central1-docker.pkg.dev/k8s-staging-images/gateway-api-inference-extension/charts/inferencepool \
+          --version v1.0.1
+          echo "---------------------------------------" >> ~/wide-ep-deployment.log
+
+      - name: Deploy Gateway and HTTPRoute
+        run: |
+          cd guides/wide-ep-lws
+          echo "Deploying Gateway and HTTPRoute..."
+          kubectl apply -k ./manifests/gateway/gke-l7-regional-external-managed -n ${NAMESPACE} | tee ~/wide-ep-deployment.log
+          echo "---------------------------------------" >> ~/wide-ep-deployment.log
+
+      - name: Wait for all pods to be ready
+        run: |
+          kubectl wait pod \
+            --for=condition=Ready \
+            --all \
+            -n "${NAMESPACE}" \
+            --timeout=90m
```

90 minutes seems pretty excessive, I have never seen a wide-ep deployment take more than 20 minutes. Maybe lets start with 25?

```suggestion
            --timeout=25m
```

---

When I was testing, it was taking closer to 30 minutes. So I updated it to 35.